srcpkgs/pulseeffects/template

```diff
@@ -1,20 +1,18 @@
 # Template file for 'pulseeffects'
 pkgname=pulseeffects
-version=5.0.3
+version=5.0.4
 revision=1
 build_style=meson
 hostmakedepends="itstool pkg-config gettext"
 makedepends="boost-devel glib-devel gsettings-desktop-schemas-devel
  gst-plugins-bad1-devel gtkmm-devel libebur128-devel lilv-devel
- pipewire-devel python3-gobject-devel sratom-devel zita-convolver-devel
- libsndfile-devel libbs2b-devel"
-depends="calf gsettings-desktop-schemas gst-plugins-bad1
- gst-plugins-good1 pipewire python3-gobject python3-scipy zita-convolver
- gstreamer1-pipewire"
+ pipewire-devel sratom-devel zita-convolver-devel
+ rubberband-devel libsamplerate-devel libsndfile-devel"
```

Comment:
why did you remove libbs2b-devel? I don't think rubberband-devel is needed, could you double check?

Reply:
@paper42 I've removed `libbs2b` because `gst-plugins-bad1` already depend on it, and also Arch package doesn't specify it as a dependency.
```
$ xbps-query -RX libbs2b
audacious-plugins-4.1_4
gst-plugins-bad1-1.18.4_3
ladspa-bs2b-0.9.1_3
libavfilter-4.3.2_1
libbs2b-devel-3.1.0_3
qmmp-1.4.6_1
audacious-plugins-4.1_4
gst-plugins-bad1-1.18.4_3
ladspa-bs2b-0.9.1_3
libavfilter-4.3.2_1
libbs2b-devel-3.1.0_3
qmmp-1.4.6_1
```
`rubberband` is optional, I've removed it.

Reply:
You removed `libbs2b-devel`, its always better to be explicit about dependencies that are directly used to avoid problems when another dependency drops it.

Reply:
> `rubberband` is optional, I've removed it.

just a small note for other reviewers, rubberband is an optional runtime dependency, it's not needed for the build

Reply:
> You removed `libbs2b-devel`, its always better to be explicit about dependencies that are directly used to avoid problems when another dependency drops it.

I've put it back